Sources/Helpers/OsmOAuthView.swift

```diff
@@ -0,0 +1,150 @@
+//
+//  OsmOAuthView.swift
+//  OsmAnd Maps
+//
+//  Created by nnngrach on 30.05.2023.
+//  Copyright © 2023 OsmAnd. All rights reserved.
+//
+
+import SwiftUI
+import Foundation
+import AuthenticationServices
+
+@available(iOS 16.4, *)
+struct OsmOAuthView: View {
+    
+    @Environment(\.webAuthenticationSession) private var webAuthenticationSession
```

Comment:
Please create a separate viewModel calss to handle the actual requests and use ObservedObjects to bind to the data in the view